dash/frontend/src/app/modules/private/private.component.html

```diff
@@ -30,31 +30,38 @@
     </div>
   </mat-toolbar-row>
   <mat-menu #menu="matMenu">
-    <button mat-menu-item>
+    &nbsp;
```

Comment:
There are lots of `&nbsp;`s added to this file - can we remove them?

Reply:
This is for the extra space requested for the top-right-dropdown menu.

Reply:
The extra space is a problem, not the end goal ;)